SolrNet/Impl/SolrServer.cs

```diff
@@ -314,9 +314,9 @@ public SolrSchema GetSchema(string schemaFileName)
             return basicServer.GetSchema(schemaFileName);
         }
 
-        public IEnumerable<ValidationResult> EnumerateValidationResults()
+        public IEnumerable<ValidationResult> EnumerateValidationResults(String schemaFileName)

```

Comment:
This method is an implementation of `ISolrOperations`, so 
- add `IEnumerable<ValidationResult> EnumerateValidationResults(string schemaFileName);` to ISolrOperations

It also shouldn't be a breaking change, so keep the non-parameterized method in place.